### ISSUE TITLE
Cyclic dependency error with Unity Jobs package

### ISSUE THREAD

**Issue by ChristianWiele** [Unity-Technologies/com.unity.perception#101]
Hi,

installing both, the Unity Perception and Jobs package (both 0.5 preview versions, installed on 2020.1.10) leads to a cyclic dependency error. As a consequence, Perception is not working.

Regards, Christian

**Comment by JonathanHUnity** [comment]
Christian, Thanks for the report. Perception currently depends on an older version of com.unity.entities, which itself depends on a 0.2 version of Jobs. I was able to reproduce your issue. A workaround is to use the current latest version of each, Jobs 0.7.0-preview.17 and com.unity.entities 0.16.0-preview.21. I ran a quick smoke test using 2d bounding boxes and perception appeared to be working properly. We will add a ticket on our side to update our dependencies to the latest version compatible with 2019.4 in the next release.

Thanks for the report, and let us know if it works.
